cleaner.go

```diff
@@ -232,6 +248,49 @@ func (c *Cleaner) setLaunchTemplateUsed() error {
 	return nil
 }
 
+// If the passed imageAlias is an alias, It will return the resolved imageId.
+// Otherwise, it will return the original imageId.
+// See https://docs.aws.amazon.com/autoscaling/ec2/userguide/using-systems-manager-parameters.html to understand this use case.
+func (c *Cleaner) resolveImageAlias(imageAlias string, launchTemplateId *string, launchTemplateVersion *int64) (string, error) {
+	// If we need to resolve aliases. We need to perform an additional DescribeLaunchTemplateVersions
+	// for each version we want to resolve. Because when calling the DescribeLaunchTemplateVersions operation: Resource aliasing (resolveAlias)
+	// is only supported when doing single version describe
+	imageIdIsAliased := strings.HasPrefix(imageAlias, "resolve:ssm:")
+	resolvedImageId, isAliasResolved := c.resolvedAliases[imageAlias]
+	imageId := imageAlias
+
+	if imageIdIsAliased && isAliasResolved {
+		// If already resolved, Just use the resolved value.
+		imageId = resolvedImageId
+	} else if imageIdIsAliased && !isAliasResolved {
+		// If have not been already resolved, resolve the alias.
+		aliasedVersions, err := c.ec2conn.DescribeLaunchTemplateVersions(&ec2.DescribeLaunchTemplateVersionsInput{
+			LaunchTemplateId: launchTemplateId,
+			Versions: []*string{
+				aws.String(strconv.FormatInt(*launchTemplateVersion, 10)),
```

Comment:
This conversion seems unnecessary.

Reply:
I could say it even looks ugly. However, it's neccesary because the `DescribeLaunchTemplateVersions` version parameter is expected to be a *string. But the `launchTemplateVersion` variable is of type `*int64`. I didn't come up with a better way to make the conversion.

Suggestions are welcome.

Reply:
Sorry, I misunderstood that `launchTemplateVersion` was a `*string` type. If it was an `*int64` type, this would make sense.